zhaquirks/tuya/ts0601_illuminance.py

```diff
@@ -0,0 +1,124 @@
+"""Tuya HOME-LUX illuminance sensor."""
+
+import math
+from typing import Dict
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Ota, Scenes, Time
+from zigpy.zcl.clusters.measurement import IlluminanceMeasurement
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import TuyaLocalCluster
+from zhaquirks.tuya.mcu import DPToAttributeMapping, TuyaDPType, TuyaMCUCluster
+
+TUYA_BRIGHTNESS_LEVEL_DP = 0x01  # 0-2 "Low, Medium, High"
+TUYA_ILLUMINANCE_DP = 0x02  # [0, 0, 3, 232] illuminance
+
+
+class BrightnessLevel(t.enum8):
+    """Brightness level enum."""
+
+    LOW = 0x00
+    MEDIUM = 0x01
+    HIGH = 0x02
+
+
+class TuyaIlluminanceMeasurement(IlluminanceMeasurement, TuyaLocalCluster):
+    """Tuya local IlluminanceMeasurement cluster."""
+
+    attributes = IlluminanceMeasurement.attributes.copy()
+    attributes.update(
+        {
+            0xFF00: ("brightness_level", BrightnessLevel, True),
```

Comment:
Maybe `manufacturer_brightness_level` here?

Must be the same value as in `dp_to_attribute` mapping.